book/src/configuration.md

```diff
@@ -119,6 +119,7 @@ the default file-picker configuration below are IgnoreOptions: whether hidden
 files and files listed within ignore files are ignored by (not visible in) the
 helix file picker and global search. There is also one other key, `max-depth`
 available, which is not defined by default.
+All git related options are only enabled in a git repository.
```

Comment:
Tiny spacing nit: can you add a blank line between these two paragraphs? They're rendered as one paragraph without the separating blank line

Reply:
Sure thing